src/process/sys/linux/process.rs

```diff
@@ -133,10 +133,10 @@ impl Process {
 				// TODO: fix or document the unwraps
 				let fd = path
 					.file_name()
-					.unwrap()
+					.ok_or(ProcessError::ParserError { pid: self.pid })?
 					.to_string_lossy()
 					.parse::<u32>()
-					.unwrap();
+					.map_err(|_| ProcessError::ParserError { pid: self.pid })?;
```

Comment:
If the filename is not a number (if that's even possible), I feel like it might be better to just ignore the file than add the error to the returned list. Or what do you think?